articles/extensions/troubleshoot.md

```diff
@@ -12,14 +12,33 @@ useCase:
 ---
 # Troubleshoot Extensions
 
-If you see issues with your [extensions](/extensions), we recommend that you begin the troubleshooting process by deleting and reinstalling the extension:
+If you see issues with your [extensions](/extensions), we recommend that you begin the troubleshooting process with the following two steps:
+
+1. Reinstalling the extension
+2. Migrating to Node.js v12
+
+## Reinstall the Extension
+
+One of the first things you can do when running into issues with an extension is to reinstall it:
 
 1. Go to [Dashboard > Extensions](${manage_url}/#/extensions).
 2. On the **Installed Extensions** tab, delete the extension.
 3. Log out of the Auth0 Dashboard.
 4. Log back in to the Auth0 Dashboard.
 5. Reinstall and reconfigure the extension.
 
+## Migrate to Node.js v12
+
+Many Auth0 extensions rely on modules that use Node.js v12. If your tenant still uses Node.js v8, you may have issues running extensions or get errors such as:
+
+```
+Compilation failed: Cannot find module 'auth0-source-control-extension-tools'
+```
```

Comment:
As it stands right now, we've made sure that all the packages that are available in Node 8 are also available in Node 12, so we _shouldn't_ run into this case. I'll talk it over with the team to see if we feel like it's still worth keeping this in there.

Reply:
Yes, I wouldn't put this information here. At this point, if they got that error, it will be another issue outside of the node 12 upgrade.

Reply:
Yeah, I agree that we can remove the example error above.